java/src/org/openqa/selenium/remote/service/DriverFinder.java

```diff
@@ -14,11 +14,14 @@ public class DriverFinder {
 
   public static String getPath(DriverService service, Capabilities options) {
     Require.nonNull("Browser options", options);
-    String defaultPath = new ExecutableFinder().find(service.getDriverName());
```

Comment:
This line is the one that finds the driver on the PATH, and it is getting removed. Right?

Reply:
no, it moved to line 20. I moved it after `getProperty()` with a conditional to only execute if there was no property found.

Reply:
You are right.